lib/CustomFields/utils/useCustomFieldsFetch.js

```diff
@@ -14,7 +14,7 @@ const useCustomFieldsFetch = (okapi, backendModuleId, entityType) => {
 
   useEffect(() => {
     const fetchCustomFields = async () => {
-      const response = await makeOkapiRequest('custom-fields')({
+      const response = await makeOkapiRequest('custom-fields?limit=2147483647')({
```

Comment:
No magic numbers please!

1. use a constant, e.g. `MAX_RECORDS`
1. use a comment explaining where that value comes from, e.g. 
`// Okapi's max record-set size` or 
`// max value as requested in STSMACOM-370`